modules/rosa-sdpolicy-am-aws-compute-types.adoc

```diff
@@ -224,8 +224,37 @@
 
 &#135; This instance type provides 48 logical processors on 24 physical cores.
 ====
+.Accelerated computing
+[%collapsible]
+====
+- p3.2xlarge (8 vCPU, 61 GiB)
+- p3.8xlarge (32 vCPU, 244 GiB)
+- p3.16xlarge (64 vCPU, 488 GiB)
+- p3dn.24xlarge (96 vCPU, 768 GiB)
+- p4d.24xlarge (96 vCPU, 1,152 GiB)
+- g4dn.xlarge (4 vCPU, 16 GiB)
+- g4dn.2xlarge (8 vCPU, 32 GiB)
+- g4dn.4xlarge (16 vCPU, 64 GiB)
+- g4dn.8xlarge (32 vCPU, 128 GiB)
+- g4dn.12xlarge (48 vCPU, 192 GiB)
+- g4dn.16xlarge (64 vCPU, 256 GiB)
+- g4dn.metal (96 vCPU, 384 GiB)
+- g5.xlarge (4 vCPU, 16 GiB)
+- g5.2xlarge (8 vCPU, 32 GiB)
+- g5.4xlarge (16 vCPU, 64 GiB)
+- g5.8xlarge (32 vCPU, 128 GiB)
+- g5.16xlarge (64 vCPU, 256 GiB)
+- g5.12xlarge (48 vCPU, 192 GiB)
+- g5.24xlarge (96 vCPU, 384 GiB)
+- g5.48xlarge (192 vCPU, 768 GiB)
+- dl1.24xlarge  (96 vCPU, 768 GiB)&#8224;
+
 
-.Compute-optimized
+&#8224; Intel specific; not covered by Nvidia
+
+Support for the GPU instance type software stack is provided by AWS. Ensure that your AWS service quotas can accommodate the desired GPU instance types.
```

Comment:
I don't have a lot of experience with these large lists, but should this bit be out side of the collapsible block? Most of the other text like this seems to be footnotes for specific items, but IIUC this applies to the whole block.